contracts/Strategy.sol

```diff
@@ -432,14 +431,25 @@ contract Strategy is BaseStrategy {
         return false;
     }
 
-    function whitelistReward(address _rewardToken, SwapSteps memory _steps) public onlyVaultManagers {
-        rewardToken = IERC20(_rewardToken);
-        rewardToken.approve(address(balancerVault), 0);
-        rewardToken.approve(address(balancerVault), max);
-        swapSteps = _steps;
+    // for partnership rewards like TUSD or airdrops
+    function whitelistRewards(address _rewardToken, SwapSteps memory _steps) public onlyVaultManagers {
+        IERC20 token = IERC20(_rewardToken);
+        token.approve(address(balancerVault), max);
+        rewardTokens.push(token);
+        swapSteps.push(_steps);
+    }
+
+    function delistAllRewards() public onlyVaultManagers {
+        for (uint i = 0; i < rewardTokens.length; i++) {
+            rewardTokens[i].approve(address(balancerVault), 0);
+        }
+        IERC20[] memory noRewardTokens;
+        rewardTokens = noRewardTokens;
```

Comment:
does `delete` not work for `rewardTokens`? Should be able to reset it to default value like swapSteps